NanoGardener/python/framework/Steps_cfg.py

```diff
@@ -1389,6 +1389,36 @@ def prepare_VBSjjlnu_Fatjet_syst(basename, selection):
                   'import'     : 'LatinoAnalysis.NanoGardener.modules.SusyGenVarsProducer' ,
                   'module'     : 'SusyGenVarsProducer()' ,
                },
+
+## EFT VBF H->WW->2l2nu
+
+    'VBFl2EFT' : {
+                  'isChain'    : True  ,
+                  'do4MC'      : True  ,
+                  'do4Data'    : True  ,
+                  'subTargets' : ['JJHEFT','EFTGen'],
+                  },
+
+
+    'JJHEFT' : {
+                   'isChain'    : False ,
+                   'do4MC'      : True ,
+                   'do4Data'    : True ,
+                   'import'     : 'LatinoAnalysis.NanoGardener.modules.JJH_EFTVars' ,
+                   'declare'    : 'JJHEFT = lambda : JJH_EFTVars()',
+                   'module'     : 'JJHEFT()',
+                 },
+
+    'EFTGen' : {
+                     'isChain'    : False ,
+                     'do4MC'      : True ,
+                     'do4Data'    : False ,
+                     'import'     : 'LatinoAnalysis.NanoGardener.modules.EFTReweighter' ,
+                     'declare'    : 'EFTGen = lambda : EFTReweighter()',
+                     'module'     : 'EFTGen()',
+                     'onlySample' : ['VBF_H0PM_ToWWTo2L2Nu','VBF_H0PH_ToWWTo2L2Nu','VBF_H0L1_ToWWTo2L2Nu','VBF_H0M_ToWWTo2L2Nu','VBF_H0PHf05_ToWWTo2L2Nu','VBF_H0Mf05_ToWWTo2L2Nu','VBF_H0L1f05_ToWWTo2L2Nu'],
```

Comment:
We should probably also apply these to the other SM samples like POWHEG ones or is it not feasible ?

Reply:
I have added the ggF, ZH and WH AC samples to the list.  
I would have thought it was sufficient to save the MEs for the AC samples. The MEs will be used to reweight one signal hypothesis to another to increase the MC stats of the templates used in the fits. So as it is, in a given channel, we will have 6 samples which we can reweight to a given hypothesis, which seems plenty. I think for ggF production it is still fine to include the standard powheg samples, but for VBF this is more complicated. These samples are NLO and have an extra jet at gen level, so we would need to recast to the best LO topology. This can be done using tools Ulascan developed, but I dont think we necessarily need it.